src/emoji/__tests__/data-test.js

```diff
@@ -57,6 +57,20 @@ describe('getFilteredEmojiNames', () => {
     expect(getFilteredEmojiNames('qwerty', { qwerty: {} })).toEqual(['qwerty']);
   });
 
+  test('returns the emoji name for a literal single-codepoint emoji search', () => {
+    const list = getFilteredEmojiNames('🤔', {});
+    expect(list).toEqual([
+      'thinking',
+    ]);
+  });
+
+  test('returns the emoji name for a literal multi-codepoint emoji search', () => {
```

Comment:
Since we have checks on the length of `query`, if it's 2 or 3, I think it would be smart to test some ordinary, non-emoji queries with those lengths, e.g., 'sad', just to make sure their behavior is the same.